packages/core/content-releases/shared/contracts/releases.ts

```diff
@@ -100,3 +100,22 @@ export declare namespace UpdateRelease {
     error?: errors.ApplicationError | errors.ValidationError;
   }
 }
+
+/**
+ * POST /content-releases/:id/publish - Publish a release
```

Comment:
nitpick: I kind of like specifying the id name everywhere:

```suggestion
 * POST /content-releases/:releaseId/publish - Publish a release
```